tokio-tasks/src/lib.rs

```diff
@@ -1,6 +1,72 @@
+use std::future::Future;
+use std::pin::Pin;
+use std::task::Context;
+use std::task::Poll;
+use std::time::Duration;
+use tokio::time::error::Elapsed;
+use tokio::time::Timeout as TokioTimeout;
+use tracing::field;
+use tracing::Span;
+
+pub use future_ext::*;
 pub use task_map::*;
 pub use tasks::*;
 
 mod future_ext;
 mod task_map;
 mod tasks;
+
+#[tracing::instrument(name = "Sleep")]
+pub async fn sleep(duration: Duration) {
```

Comment:
I was not really sure which crate or module these functions should go in

Reply:
I guess we can keep them here. I will probably rename the crate to tokio-helpers or something like that.

Reply:
Sure. For now should I move them into a separate module, though? E.g: tokio_tasks::time

Reply:
> Sure. For now should I move them into a separate module, though? E.g: tokio_tasks::time

good idea!

Reply:
Could we rename this crate to something like `tokio-util` and modify the description then? Currently it reads:

```
description = "Control the lifetime of async tasks by leveraging Drop"
```

With some of these changes that is no longer true.

Reply:
`tokio-util` is an already existing crate - could `tokio-extras` or similar be used instead?

Reply:
`tokio-extras` sounds good!